scripts/send-schedules.gmp.py

```diff
@@ -72,10 +72,12 @@ def parse_send_xml_tree(gmp, xml_tree):
 
         ical = schedule.find('icalendar').text
 
-        timezone_abbrev = schedule.find('timezone_abbrev').text
+        timezone = schedule.find('timezone_abbrev').text
+        if timezone is None:
+            timezone = schedule.find('timezone').text
```

Comment:
```suggestion
        timezone = schedule.find('timezone').text
```

`timezone_abbrev` shouldn't be an argument in a schedule anymore afaik ... https://github.com/greenbone/gvmd/commit/d4a0fa2287b425199330b7e5671b61cdbd836fe4